src/users/entities/users.repository.ts

```diff
@@ -40,6 +41,39 @@ export class UsersRepository {
     return this.repo.find();
   }
 
+  queryBuilderWithFilters(filters: UsersFilters): SelectQueryBuilder<User> {
+    const qb = this.repo.createQueryBuilder('people');
+
+    qb.innerJoinAndSelect('people.organization', 'organization');
+    const { firstName, lastName, email, organization, role } = filters;
+
+    if (firstName) {
+      qb.andWhere('people.firstName LIKE :firstName', {
+        firstName: `${firstName}%`,
```

Comment:
```suggestion
        firstName: `%${firstName}%`,
```
We need to do a partial match regardless of where it's contained in the name. Only the sections ID could be filtered by prefix as it has an ordered structure.